### ISSUE TITLE
evonorm sample2d or evonorm batch2d

### ISSUE THREAD

**Issue by cymdhx** [digantamisra98/EvoNorm#9]
How to judge which evonorm sample2d or evonorm batch2d to use

**Comment by digantamisra98** [comment]
@cymdhx Evonorm batch2d I believe is for low batch size training and sample2d is the usual replacement for batchnorm2d.
